crypto/s2n_hmac.h

```diff
@@ -12,6 +12,8 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
+/* this file is patched by siderail, clang-format invalidates patches */
+/* clang-format off */
```

Comment:
Do we want to open an issue to track updating the patches so we could format these files too?